feat(config): add support for TypeScript

wesleycho opened the pull request:
- Add TypeScript support for `karma.conf.ts`

This should address #1727.

maksimr commented on lib/cli.js:
```diff
@@ -94,10 +94,14 @@ var processArgs = function (argv, options, fs, path) {
       configFile = './.config/karma.conf.js'
     } else if (fs.existsSync('./.config/karma.conf.coffee')) {
       configFile = './.config/karma.conf.coffee'
+    } else if (fs.existsSync('./config/karma.conf.ts')) {
```
Here should be `.config`

dignifiedquire commented:
Would be good to have some test for this.

wesleycho commented:
Any tip as to where to look for putting a test? I didn't see anything for coffeescript or livescript, and the way they are set up, it may not be so simple to set up since support is checked globally.

wesleycho commented:
Found it, added a test. I also added a config template and code to the formatters to mirror the coffeescript and livescript code, although it seems they are not used.

dignifiedquire commented:
Thanks :octocat:
